edge-util/src/Microsoft.Azure.Devices.Edge.Util/Microsoft.Azure.Devices.Edge.Util.csproj

```diff
@@ -22,6 +22,8 @@
 
   <ItemGroup>
     <PackageReference Include="App.Metrics" Version="3.0.0" />
+    <PackageReference Include="App.Metrics.Formatters.Prometheus" Version="3.2.0-dev0002" />
```

Comment:
does 3.2.0-dev0002 like a rc version? should we use 3.1.0 instead?

Reply:
yeah, good point, maybe we should.